### PR TITLE
Report rusage stats for each service

### PR DESCRIPTION
**Pull request by oserres**

### CODE REVIEW

**Review by danmanj — COMMENTED** [review]
rusage should be per node not per service.

**Comment by oserres** [comment]
Updated to report the usage per nodes and for the test sequencer

File: distbench_node_manager.cc [hunk 1]
```diff
@@ -136,20 +136,30 @@ grpc::Status NodeManager::IntroducePeers(grpc::ServerContext* context,
...
     auto log = service_engine.second->FinishTrafficAndGetLogs();
     if (!log.peer_logs().empty()) {
-      (*response->mutable_instance_logs())[service_engine.first] =
-        std::move(log);
+      (*response->mutable_service_logs()
+       ->mutable_instance_logs())[service_engine.first] = std::move(log);
     }
   }
+
+  RUsageStats rusage_stats = GetRUsageStatsFromStructs(rusage_start_test,
```
Comment (danmanj) [inline distbench_node_manager.cc:158]: maybe just assign this value and avoid the variable needing to be std::move()ed

**Review by danmanj — APPROVED** [review]
maybe avoid the std::move

**Pull request merged**